supplier_app/models.py

```diff
@@ -337,8 +337,17 @@ class Address(models.Model):
         verbose_name=_("Country"),
     )
     taxpayer = models.ForeignKey(TaxPayer, on_delete=models.CASCADE)
+    history = HistoricalRecords()
+
 
+class ContactInformation(models.Model):
+    address = models.ForeignKey(Address, on_delete=models.CASCADE)
     history = HistoricalRecords()
+    contact_person = models.CharField(max_length=100, verbose_name=_("Contact person"))
+    phone_number = models.IntegerField(verbose_name=_("Phone number"))
```

Comment:
It might be better to use other data type for phone number take a look at https://docs.djangoproject.com/en/3.0/ref/models/fields/#integerfield.

Reply:
I'll probably have problems with the version of Django. As we took I'll study the posibility of made this change